npm/ux/qsharp-circuit.css

```diff
@@ -0,0 +1,180 @@
+/* Copyright (c) Microsoft Corporation.
+   Licensed under the MIT License. */
+
+/*
+  Styles for Q# Circuits.
+  Copied and modified from the default CSS generated by
+  https://github.com/microsoft/quantum-viz.js/blob/288e0cb506ee866ecca1b414c522e2d634f7b36d/src/styles.ts
+*/
+
+/* Background color for the circuit diagram */
+.qs-circuit {
+  background: var(--vscode-editor-background, #ffffff);
```

Comment:
You'll see in qsharp-ux.css variables that it accounts not only for the VS Code theme, but also for the Jupyter Labs theme. Have you tried this in Jupyter with different themes? Any reason you're not leveraging the variables in qsharp-ux.css that already account for these?

Reply:
No, no good reason, I just really struggled with this part. I see the "common" variables in qsharp-ux.css now, but they don't cover everything. e.g. some objects in the histogram/estimates use `--vscode-button-background` , but their fallbacks aren't consistent. 

The fallbacks here are just the default colors that are declared in `quantum-viz.js`, so they look okay but they of course don't respect JupyterLab's dark theme.

I just checked Histogram and Estimates, and I think Histogram also doesn't respect the theme - so I'll try to follow Estimates as an example.

Reply:
I used whatever common variables I could, but a few things (gates) just didn't have clear counterparts in the other widgets. I verified with JupyterLab light / dark themes and a bunch of custom VS Code themes.